src/components/utils/react/TableListInfo.tsx

```diff
@@ -0,0 +1,75 @@
+    </svg>
+)
+
+const sortById = "sortById"
+const SortBy = () => (
+    <>
+        <label id={sortById} style={{margin: 0, fontWeight: 600}}>Sort by</label>
+
+        <Dropdown clearable aria-labelledby={sortById}>
+            <Option key={"foo"}>Foo</Option>
+        </Dropdown>
+    </>
+)
+
+const LayoutSwitchPure = ({layout, setLayout}: { layout: TLayout; setLayout: (newLayout: TLayout) => void }) => (
+    <Toolbar
+        style={{padding: 0}}
+        checkedValues={{layout: [layout]}}
+        onCheckedValueChange={(_, {checkedItems}) => setLayout(checkedItems[0] as TLayout)}
+    >
+        <ToolbarRadioGroup>
+            <ToolbarRadioButton name={"layout"} value={TLayout.cards} icon={<CardsIcon/>}/>
+            <ToolbarRadioButton name={"layout"} value={TLayout.table} icon={<AppsList24Regular/>}/>
+        </ToolbarRadioGroup>
+    </Toolbar>
+)
+
+type ApisTableInfoProps = {
+    // apis: TApisData,
+    layout: TLayout,
+    setLayout: Dispatch<SetStateAction<TLayout>>
+    pattern: string | undefined,
+    setPattern: Dispatch<SetStateAction<string | undefined>>
+    allowViewSwitching: boolean,
+}
+
+export const TableListInfo = ({layout, setLayout, pattern, setPattern, allowViewSwitching}: ApisTableInfoProps) => (
+    <div style={{display: "flex", justifyContent: "space-between", alignItems: "center", marginBottom: "1rem"}}>
```

Comment:
There is Stack component. It comes from fluentui/react library and it is a perfect wrapper which is already flex.